tests/Support/DbHelper.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Log\Target\Db\Tests\Support;
+
+use Yiisoft\Db\Driver\Pdo\PdoConnectionInterface;
+use Yiisoft\Db\Exception\Exception;
+use Yiisoft\Db\Exception\InvalidConfigException;
+
+use function explode;
+use function file_get_contents;
+use function trim;
+
+final class DbHelper
```

Comment:
I'd move such helper into `src`. From there I'd use it in a migration in my app:

```php
public function up()
{
    (new DbHelper())->createTable();
}

public function down()
{
    (new DbHelper())->dropTable();
}
```